src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtilsUnitTest.java

```diff
@@ -705,20 +785,389 @@ private void printReferenceBases(final Path refFile, final String contig, final
+                        Strand.NEGATIVE,
+                        FuncotatorUtils.ProteinChangeInfo.create(3362,3362,"L", "")
+                },
+                {
+                        Allele.create("TCTC", true),
+                        Allele.create("T"),
+                        10081,
+                        10081,
+                        muc16ReferenceSequence.getBaseString(),
+                        Strand.NEGATIVE,
+                        FuncotatorUtils.ProteinChangeInfo.create(3361,3361,"S", "")
+                },
+                {
+                        Allele.create("CTCTGAG", true),
+                        Allele.create("C"),
+                        10082,
+                        10081,
+                        muc16ReferenceSequence.getBaseString(),
+                        Strand.NEGATIVE,
+                        FuncotatorUtils.ProteinChangeInfo.create(3362,3362,"LS", "")
+                },
+        };
+    }
+
+    @DataProvider
+    Object[][] provideForTestGetCodingSequenceChangeString() {
+
+//        final int codingSequenceAlleleStart,
+//        final String referenceAllele,
+//        final String alternateAllele,
+//        final Strand strand,
+//        final String expected
+
+        // + Strand tests from PIK3CA
+        // - Strand tests from MUC16
+
+        // TODO: Add tests for the deletion case requiring exon start/end/allele position.
+
+        return new Object[][] {
+                // SNP
+                { 1632, "A", "T", Strand.POSITIVE, "c.1632A>T" },
+                // ONP (|ONP| > 1)
+                { 1633, "CT", "GG", Strand.POSITIVE, "c.1633_1634CT>GG" },
+                // Insertion (+ Strand)
+                { 1634, "T", "TGG", Strand.POSITIVE, "c.1634_1635insGG" },
+                // Insertion (- Strand)
+                { 43303, "A", "TCA", Strand.NEGATIVE, "c.43302_43303insTC" },
+                // Deletion (+ Strand)
+                { 1634, "TGAG", "T", Strand.POSITIVE, "c.1635_1637delGAG" },
+                // Deletion (- Strand)
+                { 43138, "TCT", "T", Strand.NEGATIVE, "c.43138_43139delTC" },
+        };
+    }
 
-        //TODO: Add tests for INDELS, not just ONPs.
+    @DataProvider
+    Object[][] provideForTestIsIndelBetweenCodons() {
 
         return new Object[][] {
-                {"N",   1,  1,  "G", "AAT",    "GGT",    "p.N1G"},
-                {"NY",  1,  2, "GN", "AATTAT", "GGTAAT", "p.1_2NY>GN"},
-                {"YY",  1,  2, "NY", "TATTAT", "AATTAT", "p.Y1N"},
-                {"NY",  1,  2, "NG", "AATTAT", "AATGGT", "p.Y2G"},
-
-                {"N",  71, 71,  "G", "AAT",    "GGT",    "p.N71G"},
-                {"NY", 71, 72, "GN", "AATTAT", "GGTAAT", "p.71_72NY>GN"},
-                {"YY", 71, 72, "NY", "TATTAT", "AATTAT", "p.Y71N"},
-                {"NY", 71, 72, "NG", "AATTAT", "AATGGT", "p.Y72G"},
+                // + Strand:
+                { 4,  4, "A",   Strand.POSITIVE, false },
```

Comment:
Might want to document the meaning of the values in a comment here

Reply:
Sure.  Fixed.